src/dmd/typesem.d

```diff
@@ -1609,13 +1609,9 @@ extern(C++) Type typeSemantic(Type t, Loc loc, Scope* sc)
 
     Type visitTraits(TypeTraits mtype)
     {
-        import dmd.traits : semanticTraits;
-
-        Type result = null;
         if (mtype.ty == Terror)
-        {
```

Comment:
again, unrelated change. Please @BBasile in the future, try to keep your diff to a minimum i.e. do not mix refactorings with commits that add value.

Reply:
aesthetic.